src/renderer/caching_shaper.rs

```diff
@@ -258,24 +315,16 @@ impl CachingShaper {
         trace!("Font changed {:?} {:?}", &font_name, &base_size);
         self.font_name = font_name.map(|name| name.to_string());
         self.base_size = base_size.unwrap_or(DEFAULT_FONT_SIZE);
-        self.font_loader
-            .load(font_name.unwrap_or_default(), false, false);
+        self.font_set = FontSet::new(font_name, &mut self.font_loader);
```

Comment:
Changing font size is now instant, and changing to different font no longer reloads Symbols/Emojis etc